portfolio-stylesheet.css

```diff
@@ -0,0 +1,663 @@
+	z-index: 10;
+	left: 0;
+	top: 0;
+	padding-top: 20%;
+}
+
+.text-wrapper:hover {
+	transition: all 0.8s ease;
+	background: rgba(0, 0, 0, 0.6);
+	opacity: 1;
+	display: block;
+}
+
+.text-wrapper:hover h4 {
+	text-align: center;
+	display: block;
+	font-size: 20px;
+}
+
+.text-wrapper:hover h5 {
+	text-align: center;
+	display: block;
+	font-size: 15px;
+	font-weight: normal;
+}
+
+.name {
+	font-size: 1.5em;
+}
+
+.portfolio-page {
+	background-color: #f0f2f2;
+	padding: 0 20px 40px 20px;
+}
+
+/* ========================================================================
+	Contact Page
+   ========================================================================
+*/
+
+.contact-header {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	flex-direction: column;
+	background-color: #f7f7f7;
+}
+
+.contact-header h1 {
+	font-size: 50px;
+	font-family: "DIN Next LT";
+}
+
+.contact-image img {
+	width: auto;
+	height: 300px;
+}
+
+.top-contact-container {
+	display: flex;
+	justify-content: center;
+	align-items: flex-start;
+	background-color: #f7f7f7;
+}
+
+.top-contact-container .contact-image {
+	padding-right: 30px;
+}
+
+.top-contact-container .contact-intro-text {
+	padding: 0px 0 0 30px;
+	font-family: "DIN Next LT";
+	font-size: 20px;
+}
+
+.secondcontact-header {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	flex-direction: column;
+	background-color: #f7f7f7;
+	padding-top: 30px;
+	text-align: center;
+}
+
+.secondcontact-header h2 {
+	font-size: 35px;
+	font-family: "DIN Next LT";
+}
+
+.info-container {
+	display: flex;
+	justify-content: center;
+	text-align: center;
+	background-color: #f7f7f7;
+}
+
+.info-container h2 {
+	font-family: "DIN Next LT";
+	font-size: 25px;
+	padding-bottom: 30px;
+}
+
+.info-container {
+	display: block;
+}
+
+.contact-info h2 {
+	margin: 0;
+}
+
+
+.social-images {
+	padding-bottom: 50px;
+}
+
+/* ========================================================================
+	Universal for all pages (bottom)
+   ========================================================================
+*/
+
+.before-footer {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+/*.button.gihub:hover {
+	transform: scale(1.3);
+	color: #cdffd8;
+}*/
+
+.before-footer h6 {
+	font-family: "DIN Next LT";
+	font-size: 15px;
+	font-weight: normal;
+	margin: 0;
+}
+
+footer {
+	background-color: #e1e1e3;
+	font-family: "DIN Next LT";
+	text-align: center;
+	padding: 15px 0 15px 0;
+}
+
+/* =============================================
+	Media Queries
+   =============================================
+*/
+
+@media screen and (max-width: 1350px) {
+	.pageTitle {
+		font-size: 45px;
+	}
+}
+
+@media screen and (max-width: 1270px) {
+	.gallery-container {
+		grid-gap: 5px;
+		grid-template-columns: 1fr 1fr;
+	}
+}
+
+@media screen and (max-width: 1100px) {
+	.middle-aboutme .about-grid {
+		grid-template-columns: 1fr 1fr 1fr;
+	}
+}
+
+@media screen and (max-width: 1080px) {
```

Comment:
we tend to group media queries that are close in size together, theres not much difference between 1080 and 1100, so just use 1 media querie at 1100 for both